### ISSUE TITLE
Add IPv6 support for Kubernetes

### ISSUE THREAD

**Issue by pmichali** [contiv/netplugin#829]
As first step, we would like to have dual-stack IPv6 support for the Contiv plugin. The Kubernetes community is pushing towards supporting IPv6 and the goal is to ensure the Contiv plugin has corresponding support for IPv6.

In the future, detail tasks will be added, as they are identified.

Ref: https://github.com/kubernetes/kubernetes/issues/1443

/cc @vijaygkrishnan @shmcfarl

**Comment by pmichali** [comment]
This part of the conversation will track individual tasks/work streams for the effort.

- [ ] Program the IPv6 address and default route for a pod created under a dual-stack configuration.
- [ ] Provide host <--> pod communication over IPv6 network.
- [ ] Add IPv6 support for both L2 and L3 networking modes.

**Comment by pmichali** [comment]
@gkvijay Right, but even with L2, when IPv6 is selected, no IPv6 is set up in the pod and no default route is created (like is done for V4).  That is the first checkbox item above, and is a quick fix, IMHO.

The thought is that, we could use these comments to raise specific issues related to IPv6 support, and then commits could be applied to this issue, to address any issues that require modifications.

**Comment by pmichali** [comment]
Can we get this overall issue (or the checkbox items) prioritized?

**Comment by danehans** [comment]
@gkvijay any updates on adding IPv6 support to Contiv? CNI added IPv6 support in 0.6.0. Kubernetes plans to support CNI 0.6.0 and overall IPv6 support (alpha) in 1.9. It would be great to have Contiv listed as a supported IPv6 CNI plugin.

cc @shmcfarl

**Comment by danehans** [comment]
CNI 0.6.0 is a dep: https://github.com/contiv/netplugin/issues/967

**Comment by vhosakot** [comment]
PRs submitted to upgrade CNI to `0.6.0` in contiv:

**contiv/netplugin** - https://github.com/contiv/netplugin/pull/1100
**contiv/install** - https://github.com/contiv/install/pull/314
